cmd/solarwinds-otel-collector/components_k8s.go

```diff
@@ -116,6 +117,7 @@ func components() (otelcol.Factories, error) {
 		memorylimiterprocessor.NewFactory(),
 		batchprocessor.NewFactory(),
 		k8seventgenerationprocessor.NewFactory(),
+		swok8sworkloadtypeprocessor.NewFactory(),
```

Comment:
This processor is currently registered only in the `k8s` distro image. Do we want to include it in the main distro, too? Or should we do it later?

Reply:
Yes, main distro should have all the capabilities, so you should register it there too

Reply:
changed